dags/openshift_nightlies/config/install/alibaba/ovn-large-cp.json

```diff
@@ -21,7 +21,7 @@
     "openshift_worker_root_volume_iops": 0,
     "machineset_metadata_label_prefix": "machine.openshift.io",
     "openshift_infra_node_instance_type": "ecs.g6.8xlarge",
-    "openshift_workload_node_instance_type": "ecs.g6.4xlarge",
+    "openshift_workload_node_instance_type": "ecs.g6.8xlarge",
```

Comment:
32cpu x 132 GB required for workload node? Larger than others that are 8x32.

Reply:
Good catch, fixed!